src/js/button.js

```diff
@@ -7,13 +12,13 @@
  * @class
  * @constructor
  */
-vjs.Button = vjs.Component.extend({
+var Button = Component.extend({
   /**
    * @constructor
    * @inheritDoc
    */
   init: function(player, options){
```

Comment:
There's a [wiki page](https://github.com/videojs/video.js/wiki/5.0-Change-Details) started to track the 5.0 changes, and it's up to date with everything that's been merged into master so far. I think it makes sense to update it when things get merged in. (feel free to add to it)

Starting the migration plugin now and continually adding to it would probably best too, if anyone wants to kick that off.